rewrite/scripts/codeflow/asm.py

```diff
@@ -0,0 +1,89 @@
+from symbols import Syms, RAM_BASE
+
+class ASMInstruction:
```

Comment:
What's this do?

Reply:
analizes the log of instructions of one frame and displays the function calls